avocado/core/varianter.py

```diff
@@ -370,6 +370,26 @@ def map_method(self, method, *args, **kwargs):
         else:
             return []
 
+    def to_str(self, summary=0, variants=0, **kwargs):
```

Comment:
New method for a fake implementation.  Implementation makes sense, so nothing else to comment here.